src/analysis/retail/rogue/outlaw/CONFIG.tsx

```diff
@@ -31,8 +31,8 @@ export default {
   ),
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
   exampleReport:
-    '/report/wptPT3mfWavbj9KY/33-Heroic+Conclave+of+the+Chosen+-+Kill+(6:22)/13-Kuracz',
-
+    'https://www.warcraftlogs.com/reports/PVf9yQjhGDzZaBmv#fight=18&type=damage-done&source=11',
```

Comment:
this should be a link to a wowa report, not a WCL report.

you can take the bit starting at `/report` from your local when testing